### PR TITLE
multi-arch for tini

### PR DESCRIPTION
**Pull request by Maleware**
tini was downloaded only as amd64 binary, this was changed in the Dockerfile.

It's currently a little ugly so ideas are very welcome to make it more pretty.

### CODE REVIEW

File: pyspark-k8s/Dockerfile [hunk 1]
```diff
@@ -39,9 +39,24 @@ ENV SPARK_HOME=/stackable/spark
 ENV PATH=$SPARK_HOME:$PATH:/bin:$JAVA_HOME/bin:$JAVA_HOME/jre/bin:$HOME/.local/bin
 ENV PYSPARK_PYTHON=/usr/bin/python
 ENV PYTHONPATH=$SPARK_HOME/python:$PYTHONPATH
+ENV TINI_VERSION=v0.19.0
+
+# Ugly to download both archs here, there isn't a decent way to export an environment variable or have an if-else switch 
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-arm64 spark-k8s/stackable/utils/tini-arm64
```
Comment (adwk67) [inline pyspark-k8s/Dockerfile:45]: I don't think we pull anything directly from github in the other Dockerfiles, so we probably shouldn't do it here either: download the library to our repo and fetch it from there, to remove a dependency on github availability.

File: spark-k8s/Dockerfile [hunk 2]
```diff
@@ -32,12 +32,27 @@ ENV HOME=/stackable
 ENV JAVA_HOME=/usr/lib/jvm/jre-11
 ENV SPARK_HOME=/stackable/spark
 ENV PATH=$SPARK_HOME:$PATH:/bin:$JAVA_HOME/bin:$JAVA_HOME/jre/bin:$HOME/.local/bin
+ENV TINI_VERSION=v0.19.0
+
+# Ugly to download both archs here, there isn't a decent way to export an environment variable or have an if-else switch 
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-arm64 spark-k8s/stackable/utils/tini-arm64
```
Comment (adwk67) [inline spark-k8s/Dockerfile:38]: ditto above comment about pulling it from our Nexus.

**Review by adwk67 — CHANGES_REQUESTED** [review]

File: pyspark-k8s/Dockerfile (hunk 1 above)
Comment (Maleware) [reply pyspark-k8s/Dockerfile]: Good point! Changed that

File: pyspark-k8s/Dockerfile [hunk 3]
```diff
@@ -40,8 +40,10 @@ ENV PATH=$SPARK_HOME:$PATH:/bin:$JAVA_HOME/bin:$JAVA_HOME/jre/bin:$HOME/.local/b
 ENV PYSPARK_PYTHON=/usr/bin/python
 ENV PYTHONPATH=$SPARK_HOME/python:$PYTHONPATH
 
+# according to arch, copy binary to the name "tini"
+RUN curl -o /usr/bin/tini https://repo.stackable.tech/repository/packages/tini/tini-$(arch) 
+
 COPY spark-k8s/stackable /stackable
-COPY spark-k8s/stackable/utils/tini-v0.19.0 /usr/bin/tini
```
Comment (adwk67) [inline pyspark-k8s/Dockerfile:44]: We can delete `pyspark-k8s/stackable/utils/tini-v0.19.0` from the project

File: spark-k8s/Dockerfile [hunk 4]
```diff
@@ -33,11 +33,13 @@ ENV JAVA_HOME=/usr/lib/jvm/jre-11
 ENV SPARK_HOME=/stackable/spark
 ENV PATH=$SPARK_HOME:$PATH:/bin:$JAVA_HOME/bin:$JAVA_HOME/jre/bin:$HOME/.local/bin
 
+# according to arch, copy binary to the name "tini"
+RUN curl -o /usr/bin/tini https://repo.stackable.tech/repository/packages/tini/tini-$(arch) 
+
 COPY spark-k8s/stackable /stackable
-COPY spark-k8s/stackable/utils/tini-v0.19.0 /usr/bin/tini
```
Comment (adwk67) [inline spark-k8s/Dockerfile:37]: We can delete `spark-k8s/stackable/utils/tini-v0.19.0` from the project

**Review by adwk67 — APPROVED** [review]
lgtm (just the minor cleanup of the files in utils).

**Pull request merged**